Packages/com.unity.inputsystem/Documentation~/Migration.md

```diff
@@ -3,15 +3,25 @@ uid: input-system-migration
 ---
 # Migrating from the old Input Manager
 
+- [Read the introductory documentation first](#read-the-introductory-documentation-first)
+- [Which system is enabled?](#which-system-is-enabled)
+- [List of corresponding API in the old Input Manager new Input System package](#list-of-corresponding-api-in-the-old-input-manager-new-input-system-package)
+  - [Action-based input](#action-based-input)
+  - [Directly reading Gamepad and Joystick controls](#directly-reading-gamepad-and-joystick-controls)
```

Comment:
Why not just have "Gamepad and Joystick" ( as you have Keyboard or Mouse below)

Reply:
because it's specifically to do with the "direct" workflow - i.e. not using actions and bindings